app/views/admin/tasks/student_loan_amount.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for(:page_title) { page_title("Claim #{@claim.reference} student loan amount check for #{policy_service_name(@claim.policy.routing_name)}") } %>
```

Comment:
Looks like this gets added as a blank file in the commit "ClaimCheckingTask to model checking tasks". I think you probably meant to add it in a later commit instead.